### ISSUE TITLE
ERROR - Graph returned an error: Error validating verification code

### ISSUE THREAD

**Issue by matalla** [Automattic/facebook-instant-articles-wp#323]
When I try to connect to my page using FB login I have the following error: ERROR - Graph returned an error: Error validating verification code. Please make sure your redirect_uri is identical to the one you used in the OAuth dialog request.

Also please check this screenshot: https://www.dropbox.com/s/1sy95dnedho7sob/Screenshot%202016-06-16%2013.35.18.png?dl=0

I already set the auth redirect url to my site url but still having this error. Can you please help me fix this ?

**Comment by FlorianRachor** [comment]
I have the exact same problem. Did anybody find a solution for this? I setup the plugin on multiple pages now, this one is the first to show this error, the others worked.

**Comment by everton-rosario** [comment]
@matalla @FlorianRachor Make sure the URL you have set into the Application matches your blog's URL and the Page.

**Comment by FlorianRachor** [comment]
Well thanks for just closing this without further comment. Unfortunately your solution doesn't work, since I already did that. As I mentioned, I set this up several times, and the settings for this instance are exactly the same as for the others. So the URL settings in the APP are exactly the same as the site I'm using, and I still get the error.

**Comment by everton-rosario** [comment]
@FlorianRachor Can you share your URL/page so I can check the config issue related to your case?
Also verify that:
- your open graph meta-info content on the URL you claimed contais exactly the same info
- you created an Application and added an Web login endpoint
- The person who is logging in on Facebook and connecting the page, have the admin role for the page.

**Comment by sundarocs** [comment]
Hi,
I am getting the same error with this instant article plugin.
"_Error validating verification code. Please make sure your redirect_uri is identical to the one you used in the OAuth dialog request_"
Actual redirect_uri is https://mydomain.com/wp-admin/admin.php?page=instant-articles-wizard. Facebook app website URL is just https://mydomain.com/. 
Even I tried the same redirect_uri, but no luck. Can any one please help on this issue?

**Comment by bjackson** [comment]
Any fix for this? I have this issue as well.

**Comment by sundarocs** [comment]
Hi @bjackson: 
Yes. If your site using https, then please add this piece of code in wp-config.php file.
**$_SERVER['SERVER_PORT'] = '443';**

**Comment by ekermawi** [comment]
anyone found a solution?
i think its an https issue

**Comment by sundarocs** [comment]
@ekermawi 
Yes. If your site using https, then please add this piece of code in wp-config.php file.
$_SERVER['SERVER_PORT'] = '443';
